CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 dev:
   - change default timestamp in logs to millisecond-precision
   - allow custom timestamp formatting in logs
+  - refactor scheduler and associated interfaces
```

Comment:
We generally don't add non-user-actionable changes in the changelog, so this can be removed.